### PR TITLE
changed the arm system to a single solenoid

### PR DESCRIPTION
**Pull request by jvdyfu**
i also commented the stop command and the tests as they were giving errors due to change in the arm subsystem

### CODE REVIEW

File: src/main/java/competition/IdealElectricalContract.java [hunk 1]
```diff
@@ -81,38 +75,39 @@ public DeviceInfo turretEncoder() {
...
     }
 
-    public boolean isCollectorArmLiftingReady() {
+    public boolean isArmReady() {
         return true;
     }
 
-    public DeviceInfo liftingCollectorArmMotor() {
-        return new DeviceInfo(5, true);
+    public DeviceInfo armMotor() {
```
Comment (aschokking) [inline src/main/java/competition/IdealElectricalContract.java]: sounds like we don't need this armMotor at all anymore? You should be able to remove it

File: src/main/java/competition/IdealElectricalContract.java [hunk 2]
```diff
@@ -81,38 +75,39 @@ public DeviceInfo turretEncoder() {
...
+        return new DeviceInfo(5, true);
     }
 
     public boolean isHangingReady() {
         return true;
     }
 
     public DeviceInfo hangerMotor() {
-        return new DeviceInfo(7, true);
+        return new DeviceInfo(6, true);
```
Comment (aschokking) [inline src/main/java/competition/IdealElectricalContract.java]: revert changing these channels, they should stay matching the electrical contract doc

**Review by aschokking — CHANGES_REQUESTED** [review]

File: src/main/java/competition/IdealElectricalContract.java [hunk 3]
```diff
@@ -21,55 +21,50 @@ public DeviceInfo rightRearDriveNeo() {
...
     }
 
 
     //Everything above sure about inversion
     //Everything below not sure about the inversion
-    public DeviceInfo intakeMotor()
-    {
-        return new DeviceInfo(24, false);
+    public DeviceInfo intakeMotor() {
+        return new DeviceInfo(29, false);
```
Comment (aschokking) [inline src/main/java/competition/IdealElectricalContract.java:48]: do we need to change this channel?

File: src/main/java/competition/IdealElectricalContract.java [hunk 4]
```diff
@@ -81,38 +76,36 @@ public DeviceInfo turretEncoder() {
...
         return true;
     }
 
     public DeviceInfo frontCollectingMotor() {
         return new DeviceInfo(6, true);
     }
+    //i think this might be the same as intakeMotor but im not sure
 
-    public boolean isHangingReady() {
+    public boolean isHangingReady() { //same as climber
```
Comment (aschokking) [inline src/main/java/competition/IdealElectricalContract.java]: what's the reason for this same as climber comment?

File: src/main/java/competition/IdealElectricalContract.java [hunk 5]
```diff
@@ -81,38 +76,36 @@ public DeviceInfo turretEncoder() {
...
         return true;
     }
 
-    public DeviceInfo hangerMotor() {
-        return new DeviceInfo(7, true);
+    public DeviceInfo hangerMotor() { //same as climber
+        return new DeviceInfo(35, true);
```
Comment (aschokking) [inline src/main/java/competition/IdealElectricalContract.java]: was this what's in the contract?

File: src/main/java/competition/IdealElectricalContract.java (hunk 3 above)
Comment (jvdyfu) [reply src/main/java/competition/IdealElectricalContract.java:48]: the actual motor ID for intakeMotor is 29, according to the electrical programming contract document

File: src/main/java/competition/IdealElectricalContract.java (hunk 4 above)
Comment (jvdyfu) [reply src/main/java/competition/IdealElectricalContract.java]: in the document, it is called climber but I named by subsystem as hanger

File: src/main/java/competition/IdealElectricalContract.java (hunk 5 above)
Comment (jvdyfu) [reply src/main/java/competition/IdealElectricalContract.java]: yes

File: src/main/java/competition/IdealElectricalContract.java (hunk 3 above)
Comment (aschokking) [reply src/main/java/competition/IdealElectricalContract.java:48]: Thanks for confirming

**Review by aschokking — APPROVED** [review]
Overall looks good, thanks for updating the channels for the contract.

I was wondering if we didn't need a reverse intake command in case we collect too many balls?

**Pull request merged**